src/python/src/setup.py

```diff
@@ -98,15 +104,23 @@
     'grpc.framework': 'grpc/framework',
 }
 
+_INSTALL_REQUIRES = (
+    'enum34==1.0.4',
+    'futures==2.2.0',
+    'protobuf==3.0.0a3'
+)
+
 setuptools.setup(
     name='grpcio',
     version='0.10.0a0',
     ext_modules=_EXTENSION_MODULES,
     packages=list(_PACKAGES),
     package_dir=_PACKAGE_DIRECTORIES,
-    install_requires=[
-        'enum34==1.0.4',
-        'futures==2.2.0',
-        'protobuf==3.0.0a3'
-    ]
+    install_requires=_INSTALL_REQUIRES,
+    setup_requires=(
+        'sphinx>=1.3',
+    ) + _INSTALL_REQUIRES,
```

Comment:
Also take another look at my suggestion of

_SETUP_REQUIRES = tuple(['sphinx>=1.3'] + list(_INSTALL_REQURES))

; I think it might fix the

Processing ./src/python/src
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-YlhtI5-build/setup.py", line 115, in <module>
        ) + _INSTALL_REQUIRES
    TypeError: cannot concatenate 'str' and 'tuple' objects

in the latest Travis run.